pkgs/applications/misc/inspectrum/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, pkgconfig, cmake, fftwFloat, qt5 }:
+
+stdenv.mkDerivation rec {
+  name = "inspectrum-${version}";
+  version = "20160103";
+
+  src = fetchFromGitHub {
+    owner = "miek";
+    repo = "inspectrum";
+    rev = "a60d711b46130d37b7c05074285558cd67a28820";
```

Comment:
i used and tested the latest version as he has yet to make a formal release.  I can move back to an earlier patch but I can't see the reason